app/assets/javascripts/code/services/google/_map.js

```diff
@@ -167,6 +169,23 @@ export const map = (
       return newMarker;
     },
 
+    drawCustomMarker: function({
+        latLng: latLng,
+        content: content,
+        colorClass: colorClass,
+        clicableObject: clicableObject
+      }) {
```

Comment:
You don't need the things after `:`. This is ok
```
drawCustomMarker: function({ latLng, content, colorClass, clicableObject })
```
You can use the name after `:` in case you want to change the name of the const.